internal/db/group.go

```diff
@@ -96,6 +97,10 @@ func (d *Database) ForEachGroup(ctx context.Context, limit, offset int, f func(*
 	defer servermon.ErrorCounter(servermon.DBQueryErrorCount, &err, string(op))
 
 	db := d.DB.WithContext(ctx)
+	if match != "" {
+		match = "%" + match + "%"
```

Comment:
perhaps assert that the `match` string does not have a `%` prefix or suffix to start with

Reply:
what is the allowed format of the `match` string?

Reply:
it's just a string coming from a route argument

Reply:
i've added a test for match cointaining `%`